addons/web/static/src/core/confirmation_dialog/confirmation_dialog.js

```diff
@@ -6,6 +6,10 @@ import { useChildRef } from "@web/core/utils/hooks";
 
 import { Component } from "@odoo/owl";
 
+export const deleteConfirmationMessage = _lt("Ready to make your record disappear into thin air? are you sure\n\
+Poof! it'll be gone forever!\n\
+\nThink twice before you click that 'Delete' button!");
+
 export class ConfirmationDialog extends Component {
```

Comment:
Are you sure?   (uppercase + question mark)
It will be fone forever!\n   (Uppercase)